testsuite/src/it/java/io/quarkus/ts/startstop/NativeDebugTest.java

```diff
@@ -0,0 +1,84 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import static io.quarkus.ts.startstop.utils.Commands.getQuarkusVersion;
+import static io.quarkus.ts.startstop.utils.Commands.getBuildCommand;
+import static io.quarkus.ts.startstop.utils.Commands.cleanTarget;
+import static io.quarkus.ts.startstop.utils.Commands.getBaseDir;
+import static io.quarkus.ts.startstop.utils.Logs.archiveLog;
+import static io.quarkus.ts.startstop.utils.Logs.checkLog;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Tag("native")
+public class NativeDebugTest {
+
+    private final Apps app = Apps.JAX_RS_MINIMAL;
+    private final MvnCmds mvnCmds = MvnCmds.NATIVE;
+
+    private static final Logger LOGGER = Logger.getLogger(StartStopTest.class.getName());
+    public static final String BASE_DIR = getBaseDir();
+
+    public void testRuntime(TestInfo testInfo, File[] searchFiles) throws IOException, InterruptedException {
+        LOGGER.info("Testing app: " + app.toString() + ", mode: " + mvnCmds.toString());
+
+        File buildLogA = null;
+        File appDir = new File(BASE_DIR + File.separator + app.dir);
+        String cn = testInfo.getTestClass().get().getCanonicalName();
+        String mn = testInfo.getTestMethod().get().getName();
+        try {
+            // Cleanup
+            cleanTarget(app);
+            Files.createDirectories(Paths.get(appDir.getAbsolutePath() + File.separator + "logs"));
+
+            // Build
+            buildLogA = new File(appDir.getAbsolutePath() + File.separator + "logs" + File.separator + mvnCmds.name().toLowerCase() + "-build.log");
+            ExecutorService buildService = Executors.newFixedThreadPool(1);
+
+            List<String> baseBuildCmd = new ArrayList<>();
+            baseBuildCmd.addAll(Arrays.asList(mvnCmds.mvnCmds[0]));
+            baseBuildCmd.add("-Dquarkus.version=" + getQuarkusVersion());
+            baseBuildCmd.add("-Dquarkus.native.debug.enabled=true");
+            List<String> cmd = getBuildCommand(baseBuildCmd.toArray(new String[0]));
+
+            LOGGER.info("Building (" + cmd + ")");
+            buildService.submit(new Commands.ProcessRunner(appDir, buildLogA, cmd, 20));
+            buildService.shutdown();
+            buildService.awaitTermination(30, TimeUnit.MINUTES);
+
+            for (File searchFile : searchFiles) {
+                assertTrue(searchFile.exists());
+            }
+            assertTrue(buildLogA.exists());
+            checkLog(cn, mn, app, mvnCmds, buildLogA);
+        } finally {
+            // Archive logs no matter what
+            archiveLog(cn, mn, buildLogA);
+            cleanTarget(app);
+        }
+    }
+
+    @Test
+    public void debugSymbolsCheck(TestInfo testInfo) throws IOException, InterruptedException {
+        testRuntime(testInfo, new File[]{
```

Comment:
Provide more context in the error message

```
[ERROR] debugSymbolsCheck{TestInfo}  Time elapsed: 155.958 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	at io.quarkus.ts.startstop.NativeDebugTest.testRuntime(NativeDebugTest.java:67)
	at io.quarkus.ts.startstop.NativeDebugTest.debugSymbolsCheck(NativeDebugTest.java:80)
```
is not sufficient

Reply:
@rsvoboda PR updated

Reply:
better now

```
[ERROR] Failures:
[ERROR]   NativeDebugTest.debugSymbolsCheck:86->testRuntime:73 File or directory: /Users/rsvoboda/git/quarkus-startstop/app-jax-rs-minimal/target/quarkus-runner.debug is missing. ==> expected: <true> but was: <false>
[INFO]
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```